### PR TITLE
initial

### PR DESCRIPTION
**Pull request by ZohaibManzoor00**
### Updated Homepage UI to resemble more of Facebook 

Screenshot (333)

### Navbar, leftside and rightside all are fixed to the screen when scrolling

Screenshot (332)

### CODE REVIEW

File: src/screens/Home/HomePageBody.tsx [hunk 1]
```diff
@@ -32,23 +32,11 @@ export const mockFriendList = [
...
-  const [openEdit, setOpenEdit] = useState(false);
-  const handleOpenEdit = () => setOpenEdit(true);
-  const handleCloseEdit = () => {
-    handleMenuClose();
-    setOpenEdit(false);
-  };
   return (
-    <Box sx={{ bgcolor: isDarkMode ? "black" : "white" }}>
+    <Box sx={{ bgcolor: isDarkMode ? "#18191a" : "white"}}>
+      <Box sx={{mr: '20px', ml: '20px'}}>
```
Comment (raylu511) [inline src/screens/Home/HomePageBody.tsx:39]: u can do mx: '20px'

File: src/shared/components/ConnectionList/ConnectionList.tsx [hunk 2]
```diff
@@ -16,58 +16,59 @@ type ConnectionListProps = {
...
-                boxProps={{ my: "10px" }}
-              />
-              <Box alignSelf="center" sx={{ mx: 2 }}>
-                <ChatIcon />
+        <Box>
+          <Box sx={{ float: "right", width: "100%", mt: '10px' }}>
+            <Typography variant="h6" fontWeight="600" textAlign="center">
+              Connections
+            </Typography>
+            {connectionList.map((connection) => (
```
Comment (raylu511) [inline src/shared/components/ConnectionList/ConnectionList.tsx:48]: this is missing a key

File: src/shared/components/ConnectionList/ConnectionList.tsx [hunk 3]
```diff
@@ -16,58 +16,59 @@ type ConnectionListProps = {
...
-              sx={{ mx: 3 }}
-            >
-              <UserProfileWidget
-                {...connection}
-                key={connection.id}
-                boxProps={{ my: "10px" }}
-              />
-              <Box alignSelf="center" sx={{ mx: 2 }}>
-                <ChatIcon />
+        <Box>
```
Comment (raylu511) [inline src/shared/components/ConnectionList/ConnectionList.tsx:43]: what's this box for?

File: src/shared/components/SharePost.tsx [hunk 4]
```diff
@@ -132,7 +132,7 @@ const SharePost = ({ profilePic, onPost }: SharePostProps) => {
             )}
           </Box>
         </Box>
-        <Divider sx={{ my: 2, mt: "20px", bgcolor: navButtons }}></Divider>
+        <Divider sx={{ my: 2, mt: "20px" }}></Divider>
```
Comment (raylu511) [inline src/shared/components/SharePost.tsx:135]: this could be self closing

**Review by raylu511 — APPROVED** [review]

**Review by emir4ik-code — APPROVED** [review]

**Pull request merged**